ruby-gem/lib/calabash-android/version.rb

```diff
@@ -1,6 +1,6 @@
 module Calabash
   module Android
-    VERSION = "0.9.9"
+    VERSION = "0.9.10-1"
```

Comment:
This is not a valid version.

`0.9.10.pre1`

I would not bump the version at all for this pull request.